src/components/common/FormInput.js

```diff
@@ -29,4 +30,17 @@ const FormInput = ({
   </div>
 );
 
+FormInput.propTypes = {
+  labelClassName: PropTypes.object,
+  inputClassName: PropTypes.object,
+  inputLabel: PropTypes.object,
+  inputType: PropTypes.object,
+  inputName: PropTypes.object,
+  inputValue: PropTypes.object,
+  inputPlaceHolder: PropTypes.object,
+  inputOnChange: PropTypes.object,
+  error: PropTypes.object,
+  errorMsg: PropTypes.object,
```

Comment:
use `shape` instead of `object`, so you can validate the types inside of them. Also you can import like `import { shape } from  'prop-types';`

Reply:
On top of Tami's comment you should use an appropriate type for each prop instead of use object for all of them.
For example, I guess  labelClassName, inputClassName, inputValue, errorMsg are strings and error seems to be a boolean, so that you can define their prop types as 
FormInput.propTypes = {
  labelClassName: string, 
  error: bool
}